vm/ubpf_jit_arm64.c

```diff
@@ -140,32 +204,40 @@ emit_addsub_immediate(struct jit_state *state, bool sixty_four, enum AddSubOpcod
 
 /* [ArmARM-A H.a]: C4.1.67: Add/subtract (shifted register).  */
 static void
-emit_addsub_register(struct jit_state *state, bool sixty_four, enum AddSubOpcode op, enum Registers rd, enum Registers rn, enum Registers rm)
+emit_addsub_register(
+    struct jit_state* state,
+    bool sixty_four,
+    enum AddSubOpcode op,
+    enum Registers rd,
+    enum Registers rn,
+    enum Registers rm)
 {
     const uint32_t reg_op_base = 0x0b000000;
     emit_instruction(state, sz(sixty_four) | (op << 29) | reg_op_base | (rm << 16) | (rn << 5) | rd);
 }
 
-enum LoadStoreOpcode {
-                             // sz    V   op
-    LS_STRB   = 0x00000000U, // 0000_0000_0000_0000_0000_0000_0000_0000
-    LS_LDRB   = 0x00400000U, // 0000_0000_0100_0000_0000_0000_0000_0000
+enum LoadStoreOpcode
+{
+    // sz    V   op
+    LS_STRB = 0x00000000U,   // 0000_0000_0000_0000_0000_0000_0000_0000
```

Comment:
This is an annoying change as the comments are intended to line up with the bit-patterns to indicate what the
bits mean.

Anyway, don't delay this PR for this I'll tidy up in a follow-on PR.